consensus/types/src/beacon_state.rs

```diff
@@ -1096,9 +1115,8 @@ impl<E: EthSpec> BeaconState<E> {
             BeaconState::Fulu(state) => Ok(ExecutionPayloadHeaderRefMut::Fulu(
                 &mut state.latest_execution_payload_header,
             )),
-            BeaconState::Gloas(state) => Ok(ExecutionPayloadHeaderRefMut::Gloas(
-                &mut state.latest_execution_payload_header,
-            )),
+            // FIXME(EIP-7732): this is only to make the code compile, needs to be written later
```

Comment:
You've written it correctly but we just have to remember to look back at all the things that call these.

```suggestion
            // TODO(EIP-7732): investigate calling functions
```

Reply:
cool cool, and I updated comment in https://github.com/shane-moore/lighthouse/pull/1/commits/973264f608270b6e519e283bfb6911006c84ef76